workspaces/ui-v2/src/pages/diffs/ReviewEndpointDiffPage/ReviewEndpointDiffPageContainer.tsx

```diff
@@ -44,7 +44,7 @@ export const ReviewEndpointDiffContainer: FC<
 
   return (
     <PageLayout AccessoryNavigation={DiffAccessoryNavigation}>
-      {!endpoint || shapeDiffs.loading ? (
+      {!endpoint || shapeDiffs.loading || newRegionDiffs.loading ? (
```

Comment:
Ok this was another bug that will allow a user to recover from this state - there's a race condition where diffDescriptions can be undefined here...